Velaptor/Content/Texture.cs

```diff
@@ -25,15 +25,14 @@ public sealed class Texture : ITexture
 
         /*NOTE:
          * This flag is for the purpose of unit testing.  This flag gets set in one location and that is
```

Comment:
SS
```suggestion
         * This flag is for the purpose of unit testing and gets set in the finalizer  location.
```

Reply:
I did some investigation and discovered that this text and the associated bool variable can be removed and is not required anymore.  So I deleted it all.